libs/service-portal/core/src/components/UserInfoLine/UserInfoLine.tsx

```diff
@@ -120,14 +120,16 @@ export const UserInfoLine: FC<React.PropsWithChildren<Props>> = ({
               <SkeletonLoader width="70%" height={27} />
             ) : renderContent ? (
               renderContent()
-            ) : (
+            ) : typeof content === 'string' ? (
               <Text
                 translate={translate}
                 color={warning ? 'red600' : undefined}
                 variant="default"
               >
                 {content}
               </Text>
+            ) : (
+              content
```

Comment:
This is a far-reaching change, just check that it's not breaking anything pls

Reply:
Its doing what it was doing before, but removing the `<Text />` from the HTML when passing JSX as props. Because it could lead to invalid HTML structure errors.